### PR TITLE
Feature/atr 536 dev autonumber diagnostic

### PR DESCRIPTION
**Pull request by SENya1990**

### CODE REVIEW

File: src/Acuminator/Acuminator.Analyzers/Resources.resx [hunk 1]
```diff
@@ -513,4 +513,16 @@
   <data name="PX1007DacProperty" xml:space="preserve">
     <value>DAC property</value>
   </data>
+  <data name="PX1019Title" xml:space="preserve">
```
Comment (SENya1990) [inline src/Acuminator/Acuminator.Analyzers/Resources.resx]: @k-s-popova  please review diagnostic messages

File: src/Acuminator/Acuminator.Analyzers/DiagnosticsShortName.resx [hunk 2]
```diff
@@ -324,4 +324,10 @@
   <data name="PX1007" xml:space="preserve">
     <value>NoXmlCommentForPublicEntityOrDacProperty</value>
   </data>
+  <data name="PX1019" xml:space="preserve">
```
Comment (SENya1990) [inline src/Acuminator/Acuminator.Analyzers/DiagnosticsShortName.resx]: @k-s-popova  please review short names for diagnostics used in suppression

Comment (k-s-popova) [reply src/Acuminator/Acuminator.Analyzers/DiagnosticsShortName.resx]: @SENya1990 , I suggest to change AutoNumberingAttribute to AutoNumberAttribute in both short names just to have the names shorter.

File: src/Acuminator/Acuminator.Analyzers/Resources.resx (hunk 1 above)
Comment (k-s-popova) [reply src/Acuminator/Acuminator.Analyzers/Resources.resx]: @SENya1990 , consider changing as follows:
PX1019Title: A DAC property field with the AutoNumber attribute must have the string type
PX1019Fix: Change the type to string
PX1020Title: For a DAC property field with the AutoNumber attribute, the string length must be greater than {0}
PX1020Fix: Make the string length sufficient for the auto-numbering

File: src/Acuminator/Acuminator.Analyzers/DiagnosticsShortName.resx (hunk 2 above)
Comment (SENya1990) [reply src/Acuminator/Acuminator.Analyzers/DiagnosticsShortName.resx]: done

File: src/Acuminator/Acuminator.Analyzers/Resources.resx (hunk 1 above)
Comment (SENya1990) [reply src/Acuminator/Acuminator.Analyzers/Resources.resx]: Done, thanks!

**Pull request merged**